libs/books/feature/src/lib/reading-list/reading-list.component.scss

```diff
@@ -40,6 +40,13 @@
   white-space: nowrap;
 }
 
+.mat-icon-button-gray {
+  color: gray;
```

Comment:
Please pick colour value from `variables.scss` file.
Same applicable to line no 47 as well.

Reply:
Done